ChangeLog.md

```diff
@@ -6,6 +6,10 @@
 * Github workflows support
 * Use ATL 50% token alpha as fallback if no TokenMagicFX present
 
+
+### Version 1.2.1
```

Comment:
Gah, yours was supposed to go in first.... Sigh. go ahead and move this into the right place

Reply:
bumping to 1.2.3